sdk/quic-definitions/src/lib.rs

```diff
@@ -16,8 +16,8 @@ pub const QUIC_TOTAL_STAKED_CONCURRENT_STREAMS: usize = 100_000;
 // forwarded packets from staked nodes.
 pub const QUIC_MAX_STAKED_CONCURRENT_STREAMS: usize = 512;
 
-pub const QUIC_MAX_TIMEOUT: Duration = Duration::from_secs(2);
-pub const QUIC_KEEP_ALIVE: Duration = Duration::from_secs(1);
+pub const QUIC_MAX_TIMEOUT: Duration = Duration::from_secs(60);
```

Comment:
I think the timeout is too high to be meaningful. Typically a leader remains a leader only for a few slots. Even when the txn landed on the server it will be discarded any way

Reply:
When a validator is leader, the timeout/keep alive are not really hit because each connection is sending traffic to the leader.

The issue covered here is when the validator is NOT leader. The (thousands of) connections remain open, and they all ping the no-longer-leader every second, creating a lot of unnecessary traffic. This is what this PR fixes.

Reply:
But you are affecting the logic when it is a leader.

Reply:
I don't understand how? When a validator is leader, all connections should be sending data (so there's no keep alive), or a minority of them might be gone (so they'll eventually time out and get closed). Timing out after 2s or 60s is the same. Sending a ping every second or every 45s is a big difference.

Reply:
hmm. After review, we already have another timeout value for stream: DEFAULT_WAIT_FOR_CHUNK_TIMEOUT which is 2 second.

This timeout QUIC_MAX_TIMEOUT is the idle connection timeout value.

Reply:
Since you are already working here, can you add a comment on this constant -- connection idle timeout value?

Reply:
Added. Feel free to send me a better comment and I can apply the change.